src/directives/tooltip/__tests__/VTooltip.spec.js

```diff
@@ -0,0 +1,204 @@
+import { mount } from '@vue/test-utils'
+
+// import { mountAttachingComponent } from '../../../utils/tests-utils'
```

Comment:
@fernandacanto Could you remove this commented line here, please?

Reply:
Sure!